### PR TITLE
HTML video size annotation

### PR DESCRIPTION
**Pull request by jkrumbiegel**
png images written out via inline html are already sized using their logical size (not final bitmap size) given `px_per_unit`. This PR adds the same for videos so that you get this look by default in environments like quarto:

image

and not this instead

image

### CODE REVIEW

File: src/recording.jl [hunk 1]
```diff
@@ -179,13 +179,18 @@ function Record(func, figlike, iter; kw_args...)
 end
 
 function Base.show(io::IO, ::MIME"text/html", vs::VideoStream)
+    scene = vs.screen.scene
+    if !(scene isa Scene)
```
Comment (SimonDanisch) [inline src/recording.jl:183]: Why not `size(screen)`?

**Review by SimonDanisch — CHANGES_REQUESTED** [review]

File: src/recording.jl (hunk 1 above)
Comment (jkrumbiegel) [reply src/recording.jl:183]: seemed to return the framebuffer size in my test

Comment (jkrumbiegel) [reply src/recording.jl:183]: image

**Comment by ffreyer** [comment]
Why change root_scene to scene? Isn't that just less descriptive?

**Pull request merged**